deps/oauth2_client/test/system_SUITE.erl

```diff
@@ -281,11 +285,8 @@ init_per_testcase(TestCase, Config0) ->
 
     case ?config(group, Config) of
         https ->
-            start_https_oauth_server(?AUTH_PORT, ?config(rmq_certsdir, Config),
+            start_https_oauth_server(?AUTH_PORT, ?config(certsdir, Config),
```

Comment:
Me neither. I thought it was added on a nested group but it is not, it's stored at the same level as `rmq_certsdir`

Reply:
It is because the certificates are generated on the most outer group called `https` .. there the test generates the certificates that RabbitMq will use and the same certificates are also used for the idp .. so, the certificates are generated in the outer `https` group however the idp server is started on a subgroup .. but the idp has to use the certificates generated on the outer `https` group, hence the test cannot use `rmq_certsdir` and instead the folder where certs are located is provided in the variable `certsdir`

Reply:
I still don’t see why `rmq_certsdir` would be missing. Something must be removing it. I mean subgroups get the `Config` returned by the parent group `init_per_group/2`.

However, I see errors in several `init_per_group/2` where `Config0` is initialized ther thrown away. I will add comments specific to that.

Reply:
GitHut ditch my comment, let’s write it again…

Reply:
So, the commit to `deps/oauth2_client/test/system_SUITE.erl` disappeared, is it expected?
 
Anyway, several `init_per_group/2` functions looked like this:
```erlang
init_per_group(Group, Config) ->
    Config0 = run_setup_steps(..., Config),
    ...
    [{property, Value}
     | Config].
```

The return value is build from `Config` (received as argument) while `Config0` was initialized and thrown away.

Reply:
@dumbbell I removed that commit and then restored it while testing a new path style for the certs. I see your point, we were losing the configuration from the `run_setup_steps`. I fixed that on my last commit, waiting on CI.

Reply:
I see, thank you!